scripts/us_ca_chhs/healthcare_facility_bed_counts/preprocess_csv_helper.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import numpy as np
+
+# This function is to return a map{county_name: geoId} given CA county name.
+def generate_dcid_for_county(counties):
+    county_to_geoID = {}
+    sorted_counties = np.sort(counties)
```

Comment:
This function is getting lucky in at least two ways.  It assumes all counties are present.  It assumes that counties numbering by geoIds follows the sorted order.  What happens when a county is renamed or one splits?

Data Commons needs a solution.  If there is a data file mapping from county name to US county FIPS code, we can use the fact that the geoIds match the US county FIPS codes, which is still hacky but not quite as hacky.  At the least, add a todo to fix, e.g.,

TODO(wh1210): Migrate to a more canonical mapping.  Perhaps rely on the resolver.